icons/redbull.svg

```diff
@@ -0,0 +1 @@
.661.534 1.398.526.148.009.312-.033.399-.127a.246.246 0 0 0 .074-.243.114.114 0 0 1 .022-.082c.082-.075.273-.082.358-.192Zm-16.758.393c.201-.116.317-.317.403-.49-.062-.096-.137-.224-.288-.172-.136.106-.28.202-.43.287-.072.196.107.442.316.375h-.001Zm8.685-.633c.115.143.115.345.259.46a.618.618 0 0 0 .488-.144c.087-.058-.029-.144-.057-.23-.087-.115-.029-.345-.23-.345-.218.05-.457-.012-.662.057 0 .087.173.087.202.202Zm5.147-.345a1.719 1.719 0 0 1-.403.057h-.028c-.17-.157.185-.47.374-.46h.086c.105.043.078.321-.029.403Zm.196-4.961c-.254-.328-.52-.687-.81-.912l-.014-.012 1.673-.408 1.531-.152.694-.047.779-.065c-1.474-.355-3.555-.018-4.923.465l-.094-.08 1.482-.57.48-.148.478-.119c-.176-.027-2.169.251-2.661.651l-.439-.359.87-.403.84-.276.618-.149.469-.087.437-.061.531-.048.667-.022 1.039.059.809.137.796.229.666.284c.86.435 1.586 1.088 2.058 2.012a.08.08 0 0 1-.076.009l-.021-.011-.044-.035-.103-.112-.016-.012a.1.1 0 0 1-.019-.011l-.523-.426-.281-.17c-.792-.432-1.762-.617-2.613-.799l.903.291.861.272.183.074c.12.053.238.111.354.177 1.466.798 2.482 2.699 2.245 4.043.003.367-.091.679-.194 1.042l.007-.005c0 .004-.039.108-.042.115.361-1.556-.146-2.361-1.052-2.824-2.2-.924-4.281.31-6.026.561a.033.033 0 0 0 0-.014l-.001-.003c-.11-.217.939-.439 1.079-.523.082-.085.223-.056.316-.132.092-.092.231-.112.354-.136.325-.166.682-.22 1.044-.227.127.019.246-.053.373-.043a.351.351 0 0 1 .189-.031c.386-.028.783-.064 1.116-.3.034-.02.154-.086.053-.099-.632-.171-.936.261-1.512-.257.009-.022-.182-.103-.209-.103-.306.06-.592.197-.909.203-.145.032-.202-.041-.347-.041-.193.046-.462.169-.66.185-.091-.017-.544.219-.553.057a.017.017 0 0 1-.01 0c-.048-.183 2.834-1.26 1.648-.535.065.056.449-.106.811-.133.406-.086.945.212 1.122-.185-.006-.065-.162-.046-.2-.02-.078.042-.161.014-.242.033-.136.016-.663-.08-.783-.072-.095-.081.531-.064.586-.074.205-.03.454-.076.66-.115a1.33 1.33 0 0 1 .275-.005c.186.043.686-.024.153-.154-.14-.027-.584-.042-.654-.019-.109.014-.579.154-.83.096-.225-.069.704-.108.593-.169-1.152-.146-2.467-.11-4.743 1.003a18.065 18.065 0 0 0-.238-.285Zm.552 3.293c-.032.056-.082.202-.144.201-.062-.001-.089.009-.086-.057-.005-.193-.037-.431-.202-.547-.375-.584-.47.148-.604.058-.08-.163.047-.347.116-.489.032-.039.044-.084.086-.115.061-.354.384-.152.46.057.198.246.431.524.403.806-.029-.029-.029.029-.029.086ZM17.51 5.17h-.039c-2.021.003-3.976.838-5.39 2.186.263.051.506.177.736.329a.494.494 0 0 1 .17-.154c1.639-.911 4.361-1.427 6.685-.891 1.612.371 3.03 1.24 3.817 2.78a.51.51 0 0 1-.011.483c.204.285.379.587.522.895-.364-2.053-1.236-3.802-3.8-4.48 1.728.418 1.858.633 2.083.782-1.342-1.24-3.504-1.671-5.69-1.3 1.258-.406 3.317-.362 4.63.419l.03.018A7.165 7.165 0 0 0 17.51 5.17Zm-5.005 5.469c-.029.029-.058.029-.115.029-.077-.29-.104-.594-.259-.834-.08-.079-.184-.182-.115-.317a.136.136 0 0 1 .173-.028c.263.196.308.471.316.805.058.057.029.23 0 .345Zm8.131-2.707.037.035-.308-.033-.233-.071-.157-.081-.046-.034-.3-.114-.032-.017c-.023-.015-.033-.026-.022-.033.302.068.839.167 1.061.348Z"/></svg>
```

Comment:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Red Bull</title><path d="M.398 15.656H.396zm0 0c-.34.14-.16.558-.372.773 0 .028-.058.086 0 .115.142.028.257.086.429.114.076-.256.15-.54.2-.773-.058-.086-.143-.2-.258-.229zm.629 0c.2-.115.315-.315.4-.487-.06-.096-.135-.223-.285-.172-.137.106-.28.201-.43.287-.071.194.107.439.315.372zm10.454.2c-.53.04-1.565.877-2.061.401-.172-.114-.23-.287-.344-.458-.114-.172-.315-.028-.458-.114-.074-.234-.44-.512-.286-.745.31-.065.677-.177.944.057.23.258.315.573.63.745.949.031 1.282-.7.459-1.26-.408-.21-.839-.57-.92-1.06-.028-.028-.114-.057-.142-.028-.115.086 0 .143 0 .257.026.61-1.246.158-1.69.286-1.296.213-1.48-.164-1.202-1.378 0-.057-.086-.086-.115-.086-.293-.015-.226.982-.343 1.232-.67 1.626-2.631.52-3.493 1.776-.041.052-.184.338 0 .287 1.201-1.377 2.766.104 3.665-1.715.35.096.072.466-.028.658-.086.172-.172.4-.373.487-1.201.782-1.191.1-2.233.372-.292.252-.596.524-1.003.573-.142-.029-.343-.143-.486 0-.39.609-.544.705-1.26.515.057-.257.086-.515.257-.716.28-.033.412-.257.545-.458.2-.115.487-.23.63-.487.115-.264.148-.52.343-.744.376.023.745-.102 1.03-.347.176-.274-.146-.576-.172-.858-.057-.258-.029-.516-.057-.802-.235-.122-.697.174-1.06.057-.982-.061-1.666-1.373-.973-2.173.278-.42.87-.655 1.088-1.09-.1-.304-.591-.095-.802-.226C1.057 8.65.657 7.986.97 7.496c.678.566 1.348.35 1.918.858.548.812-.555 1.329-1.004 1.831-.644.488-.364 1.594.459 1.633.517.028.919-.23 1.348-.487 1.601-1.429 2.042-.355 3.207-1.575.782-.685 1.187.007 1.861-.858.43-.372.945-.43 1.432-.63.6-.277 1.227-.844 1.947-.544.546.175 1.702 1.27 2.145 1.632.5.652 1.22 1.276 1.378 2.119.157.868.258 1.929.858 2.549.229.315.372.744.744.945l.259.057c.014.046.019.082-.029.115-.977.203-1.047-1.193-1.716-1.203.152.303.383.596.458.916.057.23.315.4.516.515.064.054.474.157.286.287a1.577 1.577 0 0 1-1.231-.487.38.38 0 0 0-.286-.172c-.432.585-1.052.242-1.604.459-.327.093-.827.396-1.088.028-.075-.351-.106-.428.143-.744-.013-.225-.255.065-.286-.115 0-.286.286-.43.458-.63.054-.209.152-.404.286-.573-.103-.146-.552.075-.773.029-.119.098-.703.297-.687 0 .167-.19.45-1.052.172-1.232a.129.129 0 0 0-.086.087c-.058.4-.115.715-.316 1.03-.182.242-.652.102-.744.344.037.25.497.958.602 1.203 0 .23-.086.458-.144.658.219.028.365-.371.43-.544.034-.238-.457-1.018-.545-1.231.12-.167.33-.044.487 0 .053.426.748.898.487 1.374-.157.345-.368.767-.83.716zm3.033-3.178c.133.09.228-.639.601-.057.165.115.196.351.201.544-.003.066.024.056.086.057.062.001.111-.145.143-.2 0-.057 0-.115.029-.086.028-.28-.204-.557-.401-.802-.076-.209-.397-.41-.458-.057-.042.031-.054.076-.086.114-.069.142-.194.325-.115.487zm-.143 2.062h.029a1.71 1.71 0 0 0 .4-.058c.107-.081.134-.357.03-.4h-.087c-.188-.01-.541.301-.372.458zM12.08 9.843c.154.24.181.541.258.83.057 0 .086 0 .115-.028.028-.115.057-.287 0-.344-.009-.333-.053-.606-.315-.802a.137.137 0 0 0-.173.029c-.069.134.035.236.115.315zm-2.405 5.183c.114.143.114.343.258.458a.616.616 0 0 0 .486-.143c.086-.057-.028-.143-.057-.23-.086-.114-.028-.343-.229-.343-.217.05-.455-.012-.659.057 0 .087.172.087.2.201zm4.037 2.796h.006-.006zm8.835-5.804c.06.03.09.098.073.162-.25.94-.824 1.577-1.437 2.247l.003.004c.093-.065.19-.117.286-.168.183-.097.355-.188.471-.363.651-.982.883-1.53 1.038-2.11a1.749 1.749 0 0 0-.372-.28c-.038.164-.077.323-.123.493a.14.14 0 0 1 .061.015zm1.438-.657c-.306-2.281-1.087-4.283-3.87-5.019 1.72.416 1.85.63 2.075.78-1.337-1.236-3.49-1.665-5.667-1.296 1.27-.41 3.355-.36 4.66.447-3.112-1.92-6.853-1.09-9.153 1.103.43.083.806.366 1.157.637 2.61-1.42 7.996-1.808 9.787 1.663-.103.064-.184-.11-.26-.16-.014-.005-.046-.036-.06-.043-.852-.855-2.218-1.108-3.37-1.35.769.305 1.578.398 2.298.806 1.878 1.077 2.698 3.258 2.048 5.036l.006-.005c.29-.834.408-1.718.35-2.599zm-6.258 7.44c.215-.034.43-.092.644-.145a8.687 8.687 0 0 1-.644.145zm5.835-4.735h.002l-.003.008c-.565 2.297-2.84 3.998-5.19 4.582.084-.025.157-.05.22-.081-.3.06-.3.06-.357.068.093-.03.1-.038.32-.1a2.394 2.394 0 0 1-.372.074 7.877 7.877 0 0 0 3.709-1.991c-.04.037-.08.072-.123.104.562-.548.864-.888 1.1-1.361-.462.33-.685.874-1.12 1.225.2-.238.2-.238.214-.28a1.03 1.03 0 0 1-.075-.009c0-.01.004-.022.004-.03-2.58 1.91-5.723 3.182-8.642 1.5 1.19.51 1.475.471 1.536.45-.862-.309-1.038-.389-1.067-.402 1.097.114 2.597.462 3.89-.01.82-.231 1.416-.539 1.343-.57-1.624.343-1.766.759-4.092.575l-.316-.032c-1.707-.195-2.278-.938-2.282-.946.237.074.415.268.666.311-.412-.252-.846-.636-1.2-.98 1.586 1.292 3.936 1.198 5.905.438a6.882 6.882 0 0 1-2.072-.184c-.94-.222-1.717-.626-.645-.019.65.32-1.114-.153-1.544-.526.193-.05.377-.126.565-.172.043-.012.087-.02.13-.027a.908.908 0 0 1 .206-.01c.536.082.79-.023 1.2-.288.301.172.659.531 1.393.523a.553.553 0 0 0 .35-.09c.092-.059.148-.151.12-.278-.003-.047.02-.076.051-.101.08-.05.212-.073.295-.145a.205.205 0 0 0 .09-.154c.889-.198 1.634-.638 2.063-1.327a1.226 1.226 0 0 1-.245.132c-.011.005-.705.19.403-.534.988-.589 1.06-1.328 1.501-2.39-1.813-.278-3.518.602-4.984.812v-.014c-.119-.216.933-.44 1.072-.523.082-.085.222-.056.315-.132.091-.091.23-.11.353-.135.323-.165.68-.219 1.04-.226.126.019.244-.052.37-.043a.347.347 0 0 1 .19-.03c.384-.029.778-.065 1.11-.3.035-.02.153-.085.053-.098-.63-.17-.932.26-1.506-.256.01-.022-.18-.103-.208-.103-.304.06-.59.197-.905.202-.144.032-.2-.04-.345-.04-.193.045-.46.169-.657.184-.092-.017-.542.218-.55.056a.017.017 0 0 1-.01 0c-.049-.182 2.821-1.254 1.64-.532.065.056.447-.105.807-.132.404-.086.941.21 1.118-.185-.007-.065-.162-.046-.2-.02-.077.043-.16.015-.24.033-.136.016-.66-.079-.78-.07-.095-.082.528-.065.583-.074a19.1 19.1 0 0 0 .657-.115c.091-.011.183-.013.274-.006.186.044.683-.024.153-.153-.14-.027-.581-.041-.652-.019-.108.014-.576.153-.825.095-.225-.068.7-.106.59-.168-1.148-.145-2.457-.109-4.725 1a18.203 18.203 0 0 0-.2-.242c-.27-.344-.55-.729-.856-.962 1.392-.556 3.484-.539 4.658-.67-1.469-.353-3.54-.018-4.902.463l-.094-.08c.667-.3 1.766-.7 2.429-.833-.175-.027-2.16.25-2.65.648-.102-.087-.207-.171-.31-.254 1.615-.856 4.393-1.355 6.65-.664 1.12.345 1.983.934 2.572 1.755-.834-.691-2.011-.937-3.059-1.156l-.158-.033a.126.126 0 0 0-.148.087.127.127 0 0 0 .075.155c.311.124.63.213.938.3.475.133.923.26 1.343.498 1.482.85 2.304 2.446 2.171 3.974.001.021.007.043.008.064-.008.34-.057.67-.136.99zm-4.07-6.463c-.044.026.257.134.353.163-.008-.005.24.212.741.218-.19-.202-.775-.31-1.093-.381zm.766 7.356c-.123.11-.482.25-.652.244.136.017.433-.088.596-.192.017-.018.038-.034.056-.052zm.453-1.093c.566-.519 1.083-1.146 1.144-1.912-.079.07-.085.407-.33.449-.06 0-.216-.03-.195-.103-.142.494-.576.714-.762 1.123.185.088-.125.288-.101.432a.254.254 0 0 1 .244.011zm2.437-2.098a2.27 2.27 0 0 0-.639-.495c-.024-.01-.047-.012-.07-.021l.091.036a14.66 14.66 0 0 1-.293 1.143c.103-.13.165-.17.267-.297-.243.934-.817 1.557-1.444 2.249-.168.186.243-.013-.116.373.087-.093.067-.043.134-.006.286-.36.746-.378 1.006-.771.663-1.01.9-1.578 1.064-2.21z"/></svg>
```
I used the logo available at the website header from https://www.redbull.com/us-en/